build/installers/liferay-developer-studio/liferay-developer-studio.xml

```diff
@@ -564,6 +593,25 @@
                 </fileTest>
             </ruleList>
         </parameterGroup>
+        <fileParameter>
+            <name>activationKey</name>
+            <title>Activation Key(Optional)</title>
+            <description>activation key file</description>
+            <explanation>Please select the activation key file for Liferay DXP. Or you can put it into DXP later.</explanation>
```

Comment:
` <explanation>Please select the activation key file for Liferay DXP.  If you skip this step, you can add the activation key into auto-deploy folder later.</explanation>`